tests/data/transformer-module/02-module-with-extends/combined.fga

```diff
@@ -0,0 +1,27 @@
+model
+  schema 1.2
+
+# module core, file core.fga
+type user
+  relations
+    # extended by: module board, file board.fga
+    define another_thing: [user]
+    # extended by: module org, file org.fga
+    define manager: [user]
+    define thing: [other]
+
+# module core, file core.fga
```

Comment:
This is not exactly how I was thinking about it

I was thinking they would be grouped by file name and the comment would be at the top of the top most type in that file.


But maybe we can stick to this, let's see what @aaguiarz thinks.

Reply:
Also are we grouping by file here? The code seems to indicate no

Reply:
From a keep it clean perspective I agree, however in practice might that lead to continually scrolling up to see the module/file a type originally came from? It also makes the code a lot less simple I guess.

I think the sorting needs fixing on the types too yeah my bad, I think i just need to extract the relations sorting and apply it to types. I'll maybe change the tests up a bit to try and add in more variance